WordPress/openverse
# Replace "Over..." language with more precise "Top..."

## Fixes



Follow up to https://github.com/WordPress/openverse/discussions/4476 by @obulat 

## Description


The API no longer returns an abstract, unreachable limit of 10k results. With this comment, we will show a precise number of actually reachable results, while still clarifying that these (may) be only the _top_ <x> results for that query (and not _all_ <x> results for that query).

This is one option for resolving this, as described in the linked discussion.

## Testing Instructions

Checkout the branch and run `./ov just p frontend dev` and visit http://localhost:8443. Search for something that will return heaps of results, like "birds". Now try something more niche like "big green bird" (which mostly just reduces audio results). Notice for each of these the text in the search bar and in the media type buttons says "Top <whatever> results/<media type>".

Do this for each of the three types of searches (all media, image, audio), and confirm you get the same "Top <x>" text when the results are at the "max" value for any of the individual media types (240).

## Checklist


- [N/A] I ran the DAG documentation generator (`just catalog/generate-docs` for catalog
      PRs) or the media properties generator (`just catalog/generate-docs media-props`
      for the catalog or `just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
